src/modules/Head/components/DynamicPreviewBranding.tsx

```diff
@@ -34,8 +34,7 @@ export function DynamicPreviewBranding({ settings }: Props) {
 
     useEffect(() => {
         setPreviewSettings(parsedPreviewSettings);
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [JSON.stringify(parsedPreviewSettings), setPreviewSettings]);
+    }, [setPreviewSettings, parsedPreviewSettings]);
```

Comment:
`parsedPreviewSettings` is an object that is not memoized so this effect will keep triggering. It was correct before using the `JSON.stringify`.